hawc/services/epa/hero.py

```diff
@@ -90,24 +107,53 @@ def get_content(self):
             self.failures = []
             return dict(success=self.content, failure=self.failures)
 
+        if settings.HAWC_FEATURES.ENABLE_NEW_HERO:
+            # ensure valid api key
+            headers = {"Authorization": f"Bearer {settings.HERO_API_KEY}"}
+            r = requests.get(
+                "https://heronetnext.epa.gov/api/user/check",
+                headers=headers,
+                timeout=10.0,
+            )
+            if r.status_code != 200:
+                logger.info("Valid HERO API key required.")
+                return dict(success=[], failure=self.ids)
+
         rng = list(range(0, self.ids_count, self.settings["recordsperpage"]))
         for recstart in rng:
             request_ids = self.ids[recstart : recstart + self.settings["recordsperpage"]]
             ids = ",".join([str(id_) for id_ in request_ids])
             rpp = self.settings["recordsperpage"]
-            url = f"https://hero.epa.gov/hero/ws/index.cfm/api/1.0/search/criteria/{ids}/recordsperpage/{rpp}.json"
-            try:
-                r = requests.get(url, timeout=30.0)
-                if r.status_code == 200:
-                    data = json.loads(r.text)
-                    for ref in data["results"]:
-                        self.content.append(parse_article(ref))
-                else:
+            results = []
+            if settings.HAWC_FEATURES.ENABLE_NEW_HERO:
+                url = "https://heronetnext.epa.gov/api/reference/export/json"
+                params = {"id": request_ids, "type": "hero"}
```

Comment:
switch to post for this b/c the request_ids could be long in a URL as url params